src/app_service/service/community/service.nim

```diff
@@ -641,10 +634,20 @@ QtObject:
               )
             )
 
-          # Handle name/description changes
-          if chat.name != prevChat.name or chat.description != prevChat.description or chat.color != prevChat.color or chat.emoji != prevChat.emoji:
-            var updatedChat = findChatById(chat.id, updatedChats)
-            updatedChat.updateMissingFields(chat)
+          if chat.name != prevChat.name or chat.description != prevChat.description or chat.color != prevChat.color or
+              chat.emoji != prevChat.emoji or chat.viewersCanPostReactions != prevChat.viewersCanPostReactions:
+            var updatedChat = chat
```

Comment:
Note here that I'm no longer using `updatedChats`, but the `chat` directly here, because `updatedChats` was missing a LOT of the fields, and not just community fields. It was also missing its ID and communityID.

I don't think it should cause a lot of problems to use `chat` directly. I tested editing the channel in multiple ways and they all worked.